.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: node_js
+sudo: required
+jdk: oraclejdk8
+before_script:
+  - sudo apt-get update && sudo apt-get install oracle-java8-installer
+  - npm run bootstrap
```

Comment:
Since we added the postinstall script, we should no longer need this.
https://github.com/cerner/terra-ui/commit/0c4a69fce4a9a92fcd840faab7a9ff01910ac24d